src/config/dynamic-theme-fixes.config

```diff
@@ -14746,6 +14746,15 @@ INVERT
 
 ================================
 
+tastoid.com/
+
+CSS
+.jumbotron.jumbotron-no-bg {
```

Comment:
This is weird, this site already looks correctly for me. The screenshot shows that background-image isn't being modified properly? Could you check if this issue is persistent?

Reply:
Hi,
Thank you for replying.
I tried on several browsers. The issue only occurs in Chrome, not Firefox nor Edge. Hope this helps.